src/pipelines/aggregatesIngestor.json

```diff
@@ -241,7 +241,126 @@
                         "type": "String",
                         "ordinal": 1
                     }
+                },
+                {
+                    "source": {
+                        "path": "$['senderCode']"
+                    },
+                    "sink": {
+                        "type": "String",
+                        "ordinal": 2
+                    }
+                },
+                {
+                    "source": {
+                        "path": "$['operationType']"
+                    },
+                    "sink": {
+                        "type": "String",
+                        "ordinal": 3
+                    }
+                },
+                {
+                    "source": {
+                        "path": "$['transmissionDate']"
+                    },
+                    "sink": {
+                        "type": "DateTime",
+                        "ordinal": 4
+                    }
+                },
+                {
+                    "source": {
+                        "path": "$['accountingDate']"
+                    },
+                    "sink": {
+                        "type": "DateTime",
+                        "ordinal": 5
+                    } 
+                },
+                {
+                    "source": {
+                        "path": "$['numTrx']"
+                    },
+                    "sink": {
+                        "type": "Int32",
+                        "ordinal": 6
+                    }
+                },
+                {
+                    "source": {
+                        "path": "$['totalAmount']"
+                    },
+                    "sink": {
+                        "type": "Int32",
```

Comment:
The batch service DTO allows [Long total amount](https://github.com/pagopa/rtd-ms-transaction-filter/blob/56872fb822eca6a826ef7d4e413baad2e9f94c74/api/batch/src/main/java/it/gov/pagopa/rtd/transaction_filter/batch/model/AdeTransactionsAggregate.java#L47), shouldn't it be `Int64`?

Reply:
Ok thanks, let's make it part of another task